CONTRIBUTING.md

```diff
@@ -7,74 +7,74 @@ to help speed and unify development.
 
 This project uses GitHub issues to track bugs, enhancements, and other tasks.
 
-To create a new Issue:
+To create a new issue:
 
 1. Click on the
 [Issues tab](https://github.com/hyperledger/education-cryptobunnies/issues)
-underneath the name of this repo above.
+underneath the name of this repo.
 2. Click on the
 [New Issue button](https://github.com/hyperledger/education-cryptobunnies/issues/new)
 on the right side of the page.
 3. Enter a title and description for the issue.
 4. If applicable, select a label on the right side (such as "bug").
 5. Click _"Submit new issue"_.
 
-The title should be a concise label identifying the bug or feature, while the
-_"Leave a comment"_ section below should contain a description with as much
-detail as possible. For bugs or typos this might include:
+The title should be a short phrase that identifies the bug or feature. The
+_"Leave a comment"_ section should contain a description with as much
+detail as possible. For bugs or typos, this description might include:
 
-- The specific behavior, and what behavior you expected
-- What steps someone else could take to reproduce the bug
+- Specific behavior for the bug and what behavior you expected
+- Steps someone else could take to reproduce the bug
 - Any console logs or other diagnostic information
-- Filenames and line numbers if possible (especially for typos)
+- Filenames and line numbers, if possible (especially for typos)
 
 ## Making Code Contributions
 
 Like many projects on GitHub, contributions should be submitted by creating a
-fork of the repo, making some changes there, and then submitting them back as a
+fork of the repo, making some changes there, and then submitting them as a
 Pull Request for review. You will find a detailed explanation of this workflow
-below. While we are open to any help you'd like to give, if you are looking for
-a good place to start, you might checkout the _Issues_ page and see if there is
+below. We are open to any help you'd like to give, but if you are looking for
+a good place to start, you might check out the _Issues_ page to see if there is
 anything labeled "help wanted" that you could tackle.
 
 ### Submitting Pull Requests
 
-While the fork to PR process is common, somewhat more unusual is that `master`
+While the fork-to-PR process is common, it is somewhat unusual that `master`
```

Comment:
Hmm.  I don't see the nuances that you do.  I was trying to simplify the complex grammatical structure of this phrase. How about this variant? 

"While the fork-to-PR process is common, this repo is somewhat unusual because master is _not_ the branch you should make a PR against."

Reply:
Perfect. That's the meaning I was hoping to get across.